package.json

```diff
@@ -110,10 +112,10 @@
         "uuid": "^8.3.2",
         "web-vitals": "^1.1.1"
     },
-    "pre-commit": [
-        "lint",
-        "prettier"
-    ],
+    "lint-staged": {
+        "*.{js,ts}": "npm run lint:fix",
+        "*.{js,ts,json,html,yml,md}": "npx prettier --write"
```

Comment:
issue(blocking): Our preference is that prettier and lint check during pre-commit, rather than fix during pre-commit. See #133 

I've found that having the commit fail helps me (1) better understand the codebase and changes, and (2) better configure these tools. For example, for prettier, we may want to add files to .prettierignore. For linting, we may want to adjust our rules.

Reply:
Hi maniator@, I agree with Quinn with respect to this change. It doesn't take too much effort to run `npm run prettier:fix` to fix the change after seeing the commit fail.

Reply:
Hi @maniator , I synced up with qhanam@ today, and we are still inclined towards keeping the current state of lint fix mechanism. However, we do think lint check only for `staged` files would be beneficial. Do you mind making the changes accordingly or close the PR? Thanks!

Reply:
Hi @maniator , I synced up with qhanam@ today, and we are still inclined towards keeping the current state of lint fix mechanism. However, we do think lint check only for `staged` files would be beneficial. Do you mind making the changes accordingly or close the PR? Thanks!

Reply:
Done!